e2e/next-sandbox/test/comments/with-suspense.test.ts

```diff
@@ -11,6 +11,12 @@ import {
 
 const TEST_URL = "http://localhost:3007/comments/with-suspense";
 
+// These tests sometimes fail on CI, because some operations aren't coming
+// through timely enough (or not at all) on the other end. We're still figuring
+// out what's the cause of this.
+// eslint-disable-next-line @typescript-eslint/unbound-method
+const skipOnCI = process.env.CI ? test.skip : test;
```

Comment:
When I skipped this test on CI (in #1241), apparently I forgot to make this change in this test file too, and since the test just happened to pass, I missed it. I should have updated this file as part of that PR, so including it now here, although unrelated to this change.